src/state/get-drag-impact/in-home-list.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+import type { DraggableId,
+  DroppableId,
+  DragMovement,
+  DraggableDimension,
+  DroppableDimension,
+  DragImpact,
+  DimensionFragment,
+  Axis,
+  Position,
+} from '../../types';
+import { add, subtract, patch } from '../position';
+
+// It is the responsibility of this function
+// to return the impact of a drag
+
+type Args = {|
+  pageCenter: Position,
+  draggable: DraggableDimension,
+  home: DroppableDimension,
+  insideHome: DraggableDimension[],
+|}
+
+export default ({
+  pageCenter,
+  draggable,
+  home,
+  insideHome,
+}: Args): DragImpact => {
+  const axis: Axis = home.axis;
+  const homeScrollDiff: Position = subtract(home.scroll.current, home.scroll.initial);
+  const originalCenter: Position = draggable.page.withoutMargin.center;
```

Comment:
TODO: check this position

Reply:
This is correct - the ordering of the variables was a little funny so i have cleaned that up